scripts/deployment-utils/changeMultisigOwner.ts

```diff
@@ -0,0 +1,135 @@
+import hre, { ethers } from "hardhat";
+import { read } from "./deploy";
+import multisigOwners from "../../multisig-owners.json";
+
+/**
+ * Changes the multisig owner of the `LiquidityBridgeContract` deployed on the current network to the safe wallet
+ * provided.
+ *
+ * This function validates the provided `newOwner` address, ensures ownership configuration matches
+ * expectations, and performs the transfer of ownership to the new multisig address. Additionally,
+ * it updates both the contract and proxy admin ownership.
+ *
+ * @async
+ * @param {string} newOwner - The address of the new multisig owner (Safe contract).
+ * @throws {Error} If the proxy contract is not deployed on the current network.
+ * @throws {Error} If the provided `newOwner` address is not a valid multisig Safe contract.
+ * @throws {Error} If the configuration of owners on the Safe does not match the expected configuration.
+ * @returns {Promise<void>} Resolves when the ownership transfer process is complete.
+ *
+ * @example
+ * // Change the multisig owner of the contract
+ * const newMultisigAddress = "0xNewSafeAddress";
+ * await changeMultisigOwner(newMultisigAddress);
+ */
+
+export const changeMultisigOwner = async (newOowner: string) => {
+  const network = hre.network.name;
+  console.info(`Changing multisig owner to: ${newOowner} - ${network}`);
+
+  const currentNetworkData =
+    multisigOwners[network as keyof typeof multisigOwners];
+
+  const { owners } = currentNetworkData;
+  const proxyName = "LiquidityBridgeContract";
+
+  const proxyAddress = read()[network][proxyName].address;
+  if (!proxyAddress) {
+    throw new Error(`Proxy ${proxyName} not deployed on network ${network}`);
+  }
+  console.info(`Proxy address: ${proxyAddress}`);
+
+  const safeOwners = await validateAngGetOwners(newOowner);
+  if (safeOwners.length === 0) {
+    throw new Error(
+      "Exiting... Provided Safe address is not a valid Safe contract."
+    );
+  }
+
+  validateOwners(safeOwners, owners);
+
+  console.info("Starting ownership transfer process...");
+  await transferOwnership(proxyAddress, newOowner);
+  console.log("Ownership transfer process complete!");
```

Comment:
```suggestion
  console.info("Ownership transfer process complete!");
```

Reply:
Btw, all the files are using dash-case can you rename this file so its not the only one using camelCase? Also, you're missing the main file with the call to this function, check the other scripts as reference

Reply:
Solved